### DISCUSSION TITLE
20231123 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by cgbosse** [nautilus-cyberneering/.github#70] [Meetings]
Time: 8:30 UTC (10:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by mario-nt** [comment]
### GUI

- Rebased and refactor https://github.com/torrust/torrust-index-gui/pull/313
- Opened https://github.com/torrust/torrust-index/issues/406


### Index

- https://github.com/torrust/torrust-index/issues/300#issuecomment-1814519300

### Discussions
- https://github.com/torrust/torrust-index/issues/390
- https://github.com/AudienseCo/engineering-principles
- https://github.com/orgs/nautilus-cyberneering/discussions/72
- https://github.com/nautilus-cyberneering/opensource/issues/15
- https://github.com/torrust/torrust-index/pull/395

### Training/Research

- Typescript function type
- Javascript functions

**Comment by josecelano** [comment]
### Torrust Index

Issues: 

- [Overhaul E2E Tests](https://github.com/torrust/torrust-index/issues/384)

PRs:

- [Stop containers running E2E tests when tests fail](https://github.com/torrust/torrust-index/pull/405)
- [Inject E2E DB connection info with environment variable](https://github.com/torrust/torrust-index/pull/403)
- [Use a different DB for E2E testing](https://github.com/torrust/torrust-index/pull/398)
- [Implement health check for the Index container](https://github.com/torrust/torrust-index/pull/395)

### Torrust Tracker

PRs:

- [Implement a health check for the tracker container](https://github.com/torrust/torrust-tracker/pull/512) **WIP**

### Torrust Website

PR review:

- [Add copy code button and contributors route](https://github.com/torrust/torrust-website/pull/106)
